shared/modules/selectors/smart-transactions.ts

```diff
@@ -132,7 +180,8 @@ export const getSmartTransactionsEnabled = (
 export const getIsSmartTransaction = (
   state: SmartTransactionsMetaMaskState,
 ): boolean => {
-  const smartTransactionsOptInStatus = getSmartTransactionsOptInStatus(state);
+  const smartTransactionsOptInStatus =
```

Comment:
Perhaps the variable name should be changed to match the selector name, since it represents something different now.

Reply:
Done.